pkg/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -296,6 +325,35 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
             </Checkbox>
           </Col>
         </Row>
+        <Row>
+          <Col>
+          <label>
+            Store Filter: 
+          </label>
+          <Checkbox
+            wrapperStyles={{ marginLeft: 20, display: 'inline-block' }}
+            id={`store-match-all-${id}`}
+            onChange={this.handleStoreMatchChange}
+            checked={options.storeMatches.length === 0}
+            name={"All"}
+          >
+            All 
+          </Checkbox>
+          {storeTypes.map(type => (
+            stores[type].map(store => (
+              <Checkbox
```

Comment:
We should pass a `key` prop here as we are looping over elements, not a big deal though, but it's good to follow best practices.